drivers/hmis_external_apis/lib/ac_hmis/import_clients_mci_20251120.rb

```diff
@@ -0,0 +1,342 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+require 'roo'
+
+# One-off import for HMIS Clients with MCI ID & MCI Unique ID from an Excel file.
+# Expected headers: MCI_ID, MCI_UNIQ_ID, FNAME, LNAME, DOB
+#
+# Import logic:
+# 1. Look up client by MCI Unique ID. If found, skip (do not update Name/DOB, do not add MCI ID).
+# 2. If not found, look up client by MCI ID. If found, ensure MCI Unique ID matches (do not update Name/DOB).
+# 3. If not found by either, create new client with provided name and DOB, and associate both MCI ID and MCI Unique ID.
+#
+# Usage: AcHmis::ImportClientsMci20251120.new(filename, dry_run: true).perform
+module AcHmis
+  class ImportClientsMci20251120
+    attr_reader :file_path, :dry_run
+
+    def initialize(file_path, dry_run: false)
+      @file_path = file_path
+      @dry_run = dry_run
+    end
+
+    EXPECTED_HEADERS = ['MCI_ID', 'MCI_UNIQ_ID', 'FNAME', 'LNAME', 'DOB'].freeze
+
+    def perform
```

Comment:
Add missing logic for duplicate MCI Unique IDs. We should only import one client per MCI Unique ID